python/ray/dag/compiled_dag_node.py

```diff
@@ -635,14 +656,10 @@ def exec_operation(
         Returns:
             True if the next operation should not be executed; otherwise, False.
         """
-        if op_type == _DAGNodeOperationType.READ:
-            with self._recv_stream:
-                return self._read(overlap_gpu_communication)
-        elif op_type == _DAGNodeOperationType.COMPUTE:
-            return self._compute(overlap_gpu_communication, class_handle)
-        elif op_type == _DAGNodeOperationType.WRITE:
-            with self._send_stream:
-                return self._write()
+        OpType = _DAGNodeOperationType
+        if op_type != OpType.COMPUTE:
```

Comment:
Eventually we would remove other `OpType`s. We should add a `[CL]` here to remind us.

Reply:
Removed.